internal/store/store.go

```diff
@@ -330,6 +333,21 @@ func (s *Store) GetBlueprintChanges(name string) []blueprint.Change {
 	return changes
 }
 
+func bumpVersion(str string) string {
+	v := [3]uint64{}
+	fields := strings.Split(str, ".")
+	for i := 0; i < len(fields); i++ {
+		if n, err := strconv.ParseUint(fields[i], 10, 64); err == nil {
+			v[i] = n
```

Comment:
This will panic if `len(fields) > 3`. It isn't likely but we are not doing any version validation on new blueprints therefore it's possible.

Reply:
You're completely right and it's a big deal. I've pushed a fix that ignores such version strings and added a test.